spec/controllers/admin/attachments_controller_spec.rb

```diff
@@ -145,7 +145,7 @@ module Alchemy
     end
 
     describe '#update' do
-      subject { put :update, attachment: {name: ''} }
+      subject { alchemy_put :update, {id: 1, attachment: {name: ''}} }
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.